jobs/JEVS_GLOBAL_ENS_PREP

```diff
@@ -38,16 +38,17 @@ export DCOMIN=${DCOMIN:-${DCOMROOT}}
 export COMINccpa=${COMINccpa:-$(compath.py $envir/com/ccpa/${ccpa_ver})}
 export COMINcmce=$COMINnaefs
 export COMINprepbufr=${COMINprepbufr:-$(compath.py $envir/com/obsproc/${obsproc_ver})}
-export COMINsnow=${DCOMIN}
-export COMINosi_saf=${COMINosi_saf:-$DCOMROOT}
-export COMINsst=${COMINsst:-$DCOMROOT} 
+export DCOMINsnow=${DCOMIN}
+export DCOMINosi_saf=${DCOMINosi_saf:-$DCOMROOT}
+export DCOMINsst=${DCOMINsst:-$DCOMROOT} 
```

Comment:
@BinbinZhou-NOAA What does `DCOMINsst` point to? Can we use the name of the analysis itself, instead of sst?